cmd/ftl/exiterror.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+type exitError struct {
```

Comment:
Was this supposed to be left unused?

Reply:
Yeah will be used for #5453 and I'd already written it so 🤷